krcal/map_builder/checking_functions.py

```diff
@@ -36,10 +36,11 @@ def check_if_values_in_interval(values          : np.array,
         raise AbortingMapCreation(raising_message)
 
 def get_core(nbins,rmax, rfid):
-    x      = np.linspace(-rmax, rmax, nbins)
-    xx, yy = np.meshgrid(x, x)
-    r      = np.sqrt(xx**2 + yy**2)
-    mask   = in_range(r, 0, rfid)
+    x        = np.linspace(-rmax, rmax, nbins)
+    xx, yy   = np.meshgrid(x, x)
```

Comment:
careful with `meshgrid`. It does things differently as we are used to. `meshgrid` loops over the second argument first, so for `meshgrid(np.arange(3), np.arange(4, 7)` you get `x = [0, 1, 2] y = [4, 4, 4]` for the first first three values, and we usually do it the other way around, I think. Not necessarily wrong, but it may affect things where order matters.